WordPress/Classes/Services/ShareExtensionService.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+@objc
+public class ShareExtensionService: NSObject {
+    class func configureShareExtension(oauth2Token: String, defaultSiteID: Int, defaultSiteName: String) {
+        do {
+            try SFHFKeychainUtils.storeUsername(WPAppOAuth2TokenKeychainUsername,
+                andPassword: oauth2Token,
+                forServiceName: WPAppOAuth2TokenKeychainServiceName,
+                accessGroup: WPAppGroupName,
+                updateExisting: true)
+        } catch {
+            print("Error while saving Share Extension OAuth bearer token: \(error)")
```

Comment:
This service is part of two targets (the main app and the extension) and DDLogSwift is only part of the main app target. It's a mess how we're using CocoaLumberjack right now...